### PR TITLE
Preparing to update to 2.2 SDK

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -46,7 +46,7 @@
     <MicrosoftExtensionsConfigurationPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsConfigurationPackageVersion>
     <MicrosoftExtensionsCopyOnWriteDictionarySourcesPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsCopyOnWriteDictionarySourcesPackageVersion>
     <MicrosoftExtensionsDependencyInjectionPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsDependencyInjectionPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>2.0.0</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>2.1.0-preview2-25711-01</MicrosoftExtensionsDependencyModelPackageVersion>
```
Comment (NTaylorMullen) [inline build/dependencies.props]: This is troublesome isn't it? I'd imagine the automatic update would override this.

Comment (pranavkm) [reply build/dependencies.props]: I've updated the file in Universe to use this new version (https://github.com/aspnet/Universe/commit/14888f46bca342d29767197265b9f8652a5a0277). I'm getting this out in the meanwhile to unblock further work.

**Comment by natemcmaster** [comment]
Could you also update the build tools in this repo as a part of this step to make sure it actually passes? `build -update` should update the korebuild-lock.txt file for you

**Comment by pranavkm** [comment]
Yup, I verified locally that updating it works. The two skipped tests were another curious side effect of updating to the 2.2. sdk that we would separately address.

**Comment by natemcmaster** [comment]
ok. :shipit:

**Pull request merged**